app/views/booking_requests/customer.html.erb

```diff
@@ -1,44 +1,67 @@
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Dear <%= @booking_request.name %>,</p>
-
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Thank you for requesting a
-guidance appointment with Pension Wise. We’ll confirm your appointment in the
-next 2 working days.</p>
-
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Your first choice is<br><strong
-   class="emphasize" style="font-size: 24px;"><%= @booking_request.primary_slot %></strong></p>
-
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Your second choice is<br><strong
-  class="emphasize" style="font-size: 24px;"><%= @booking_request.secondary_slot %></strong></p>
-
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Your third choice is<br><strong
-  class="emphasize" style="font-size: 24px;"><%= @booking_request.tertiary_slot
-  %></strong></p>
-
-  <p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Your appointment location is<br><strong
-  class="emphasize" style="font-size: 24px;">
-  <%= @booking_location.name_for(@booking_request.location_id) %></strong></p>
-
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Your contact number<br> <strong
-class="emphasize" style="font-size: 24px;"><%= @booking_request.phone %></strong></p>
-
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Your memorable word<br> <strong
-class="emphasize" style="font-size: 24px;"><%= @booking_request.memorable_word(obscure: true)
-%></strong></p>
-
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Your reference number<br> <strong
-class="emphasize" style="font-size: 24px;"><%=
-  @booking_request.reference %></strong></p>
-
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">To cancel or change your
-appointment, or if any of the information above isn't correct, call
- <%= @booking_location.online_booking_twilio_number %> or reply to this email.</p>
+<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px 0;font-size: 16px;line-height: 1.315789474;">
```

Comment:
That's what I was trying to achieve through my last version of this PR - using helpers for each tag to cut down on repetition - maybe i'll have a go at a half way house with just the paragraphs.